rpmbuild.c

```diff
@@ -383,12 +385,22 @@ static char * getTarSpec(const char *arg)
 
     if (!gotspec) {
     	rpmlog(RPMLOG_ERR, _("Failed to read spec file from %s\n"), arg);
-	specFile = NULL;
+    } else {
+	/* remove trailing \n */
+	tarbuf[strlen(tarbuf)-1] = '\0';
+	specFinal = rpmExpand("%{_specdir}/%{basename:", tarbuf, "}", NULL);
```

Comment:
Elegant! :smiley:

Reply:
I almost considered adding a %{rstrip:...} macro that strips any trailing whitespace (and still might) but seemed like an overkill for just this :sweat_smile: